### DISCUSSION TITLE
本机存储删除不了

### DISCUSSION THREAD

**Discussion by jenboy** [AlistGo/alist#6284] [Q&A]
游客登陆删除的文件 我想放在指定文件夹但是一直提示找不到文件 
报错信息：Rename /usr/guide/me/ceshi.txt /root/usr/del/ceshi.txt: no such file or directory  

![image](https://github.com/alist-org/alist/assets/121419174/88a7ccbc-86fb-4df0-b3a9-2b79726c6ea4)

检查了容器里这个文件/usr/guide/me/ceshi.txt  确认存在

![94bbcbfa800797f3e14d5ed38725282](https://github.com/alist-org/alist/assets/121419174/433c36a5-520d-46da-a0ea-7f5a90e1fc00)

/root/usr/del 这个路径的文件夹也是有的

![ddda31d97a2b14b14a6f758c3a80301](https://github.com/alist-org/alist/assets/121419174/e2b46614-8d8c-4415-b2e1-f7f8ffb1e740)

**Comment by jenboy** [comment]
本机存储回收站路径设置

![e8fdc339e4f7d07cdb372df0583090d](https://github.com/alist-org/alist/assets/121419174/22d21af0-75d2-49fb-bdff-4524e2bde94c)

**Comment by anwen-anyi** [comment]
Docker吗？ Docker可能得映射路径

**Reply by jenboy** [reply]
我用的docker 装的alist   要在回收站路径里 映射进去么

**Reply by jenboy** [reply]
这是挂载的路径

![image](https://github.com/alist-org/alist/assets/121419174/9d841342-ba79-4cf5-845b-3e3df66c8fd0)


![image](https://github.com/alist-org/alist/assets/121419174/5a954a65-07ec-4442-b96b-abff421767f2)


我填写宿主机挂载的路径也提示找不到


![image](https://github.com/alist-org/alist/assets/121419174/1e867295-e095-475f-9de4-83d0530b8084)

**Comment by muquew** [comment]
![PixPin_2025-01-06_14-32-31](https://github.com/user-attachments/assets/58bed08d-7e8b-4f3a-9183-42c65e92717f)
![PixPin_2025-01-06_14-33-09](https://github.com/user-attachments/assets/6969fe3b-aae5-4bff-a283-0648bbfc94b9)
解决了吗，我这里也不行

**Reply by jenboy** [reply]
没有，不知道什么原因
